cdap-app-fabric/src/main/java/io/cdap/cdap/internal/credential/handler/CredentialProviderHttpHandlerInternal.java

```diff
@@ -45,8 +47,8 @@
 @Path(Constants.Gateway.INTERNAL_API_VERSION_3)
 public class CredentialProviderHttpHandlerInternal extends AbstractHttpHandler {
 
-  private static final Gson GSON = new GsonBuilder().registerTypeAdapter(Instant.class,
```

Comment:
The returned ProvisionedCredential still has an instance of `Instant`, why do we want to remove this?

Reply:
Do you see any harm in returning an `Instant` here?

The reason I used it as `Instant`, I am able to easily deserialize the response easily here: https://github.com/cdapio/cdap/pull/15307/files#diff-d98441b91561fbd19d975fbacf290f18c6a18dfc7d7e38d2a9745894b851a3f2R200

Reply:
SG, didn't realize that Instant was already serializable/deserializable by GSON. Thanks!